src/core/formulations.jl

```diff
@@ -233,16 +233,27 @@ import PowerModels: QCRMPowerModel
 
 import PowerModels: QCLSPowerModel
 
+"""
+Abstract type for Service Formulations (a.k.a Models)
+
+# Example
```

Comment:
I see that you just moved this. But I don't know if it will parse correctly. Should be `Examples` and the julia code should be wrapped in triple backticks.